integration/tests/tracer/conftest.py

```diff
@@ -22,3 +31,246 @@ def tracer_api(tracer_json_rpc_client_session, request):
 @pytest.fixture(scope="class")
 def storage_object(web3_client, storage_contract):
     return StorageContract(web3_client, storage_contract)
+
+
+@pytest.fixture(scope="class")
+def send_neon_tx_receipt(accounts, web3_client) -> TxReceipt:
+    sender, recipient = accounts[0], accounts[1]
+    receipt = web3_client.send_neon(sender, recipient, 0.1)
+    assert receipt["status"] == 1, f"Transaction failed: {receipt}"
+    return receipt
+
+
+@pytest.fixture(scope="class")
+def call_storage_tx_receipt(accounts, storage_object):
+    sender_account = accounts[0]
+    store_value = randint(1, 100)
+    _, _, receipt = storage_object.call_storage(sender_account, store_value, "blockNumber")
+    assert receipt["status"] == 1, f"Transaction failed: {receipt}"
+    return receipt, store_value
+
+
+@pytest.fixture(scope="class")
+def send_raw_transaction_receipt(accounts, web3_client):
```

Comment:
let's delete a tests with wthh this transaction
it's something strange 
the transaction without 'to' field, with strange data

You call send_raw_transaction in each fixture under the hook